package.json

```diff
@@ -42,30 +42,28 @@
     "vue": "^3.3.x"
   },
   "devDependencies": {
-    "@types/lodash-es": "4.17.9",
+    "@rushstack/eslint-patch": "^1.4.0",
+    "@types/lodash-es": "^4.17.9",
     "@types/node": "20.6.0",
-    "@typescript-eslint/eslint-plugin": "6.7.0",
-    "@typescript-eslint/parser": "6.7.0",
     "@vitejs/plugin-vue": "4.3.4",
+    "@vitest/coverage-v8": "^0.34.4",
```

Comment:
code coverage wasn't working with `c8` but now it does with `"@vitest/coverage-v8"`